xdsl/tools/command_line_tool.py

```diff
@@ -107,7 +107,7 @@ def get_all_passes() -> list[type[ModulePass]]:
         convert_stencil_to_ll_mlir.ConvertStencilToLLMLIRPass,
         dead_code_elimination.DeadCodeElimination,
         DesymrefyPass,
-        stencil_global_to_local.GlobalStencilToLocalStencil2DHorizontal,
+        stencil_global_to_local.DistributeStencilPass,
```

Comment:
Do we need the "Pass", should it be homogeneous across the rest?

Reply:
Every class that is a pass should end in "...Pass", this makes discoverability in the codebase much better we found.